wagtailio/core/blocks.py

```diff
@@ -9,11 +9,21 @@
 from wagtail.images.blocks import ImageBlock
 from wagtail.snippets.blocks import SnippetChooserBlock
 
+from wagtail_ai.prompts import DefaultPrompt
 from wagtailmedia.blocks import VideoChooserBlock
 
 from wagtailio.core.choices import SVGIcon
 
 
+# Enable AI prompts on the alt text field of ImageBlock
+ImageBlock._meta_class.form_attrs = {
+    "data-controller": "wai-field-panel",
+    "data-wai-field-panel-main-input-value": '[data-contentpath="alt_text"] input',
+    "data-wai-field-panel-image-input-value": '[data-contentpath="image"] input',
+    "data-wai-field-panel-prompts-value": f"[{DefaultPrompt.CONTEXTUAL_ALT_TEXT}]",
+}
```

Comment:
A bit hacky but it's the most minimal change without creating a `ImageBlock` subclass in a way that won't cause new migrations